app/src/main/java/com/twilio/video/app/ui/room/ParticipantAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package com.twilio.video.app.ui.room
+
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import com.twilio.video.app.participant.ParticipantViewState
+import io.reactivex.Observable
+import io.reactivex.subjects.PublishSubject
+
+internal class ParticipantAdapter : ListAdapter<ParticipantViewState, ParticipantViewHolder>(
+        ParticipantDiffCallback()) {
+
+    private val viewHolderEventsSubject = PublishSubject.create<RoomViewEvent>()
```

Comment:
What are your thoughts on using LiveData here instead of RX?

I'm not opposed to RX, but I know we discussed potentially phasing RX out in favor of co-routines.